www/templates/goals/challenge-detail.html

```diff
@@ -0,0 +1,35 @@
+<ion-modal-view style="background-color: #ededed;">
```

Comment:
like the separate files for screens. Would be good to have matching separate js as well instead of dumping everything into `goals.js`